style.css

```diff
@@ -1,469 +1,493 @@
+    background-color: #36b37f;
+    text-decoration: none;
+  }
+
+  .button2:hover {
+    background-color: #36b37f;
+    cursor: pointer;
+    box-shadow: 0 0 5px rgba(0, 0, 0, 0.24);
+  }
+
+  .button2:active {
+    background-color: #008552;
+    box-shadow: 0 5px #666;
+    transform: translateY(4px);
+  }
+}
+
+@media only screen and (min-width: 375px) and (max-width: 1800px) {
+  /* CARD WORK LINE */
+  .about-container {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    gap: 24px;
+    margin: auto;
+    width: 1155px;
+    padding-top: 150px;
+  }
+
+  .card-about {
+    display: flex;
+    justify-content: space-evenly;
+    border-radius: 10px;
+    background-color: #ebf0ee;
+    flex-direction: column;
+    align-items: center;
+    border: 1px solid #ccc;
+    padding: 20px;
+    width: 369px;
+    height: 316px;
+  }
+
+  .card-about img {
+    height: auto;
+  }
+
+  .txt {
+    margin-top: -10%;
+  }
+
+  .card-about .txt {
+    font-size: 24px;
+    line-height: 32px;
+    margin-bottom: 10px;
+    color: #344563;
+  }
+
+  .card-about ul {
+    list-style: none;
+    padding: 0;
+    margin: 0;
+  }
+
+  .card-about li {
+    margin-bottom: 5px;
+  }
+
+  .stack {
+    display: flex;
+    justify-content: space-between;
+    gap: 24px;
+    font-size: 12px;
+  }
+
+  .items {
+    color: #36b37e;
+    padding: 8px 12px;
+    background-color: white;
+    border-radius: 2px;
+  }
+
+  .frame-p {
+    flex: 1;
+    font-family: "Inter", sans-serif;
+    font-size: 30px;
+    line-height: 52px;
+    text-align: center;
+    color: #172b4d;
+    padding-top: 60px;
+    font-weight: 700;
+    padding-right: 20px;
+  }
+
+  .frame-p p {
+    width: 463px;
+    height: 256px;
+  }
+
+  .frame-regist {
+    display: grid;
+    margin: auto;
+    grid-template-columns: repeat(2, 1fr);
+    align-items: center;
+    width: 1185px;
+  }
+
+  .frame-form {
+    padding-top: 1px;
+    position: relative;
+    right: 40px;
+  }
+
+  .names {
+    display: flex;
+    gap: 14px;
+  }
+
+  .frame-form textarea {
+    height: 114px;
+    display: flex;
+    width: 684px;
+    justify-content: end;
+    font-size: 20px;
+    font-size: 15px;
+  }
+
+  #email {
+    margin: auto;
+    border: 1px solid #d0d9d4;
+    border-radius: 4px;
+  }
+
+  .frame-form #email {
+    display: flex;
+    width: 684px;
+    height: 48px;
+    justify-content: end;
+    font-size: 15px;
+  }
+
+  #lastname {
+    margin: auto;
+    border: 1px solid #d0d9d4;
+    border-radius: 4px;
+  }
+
+  .frame-form #lastname {
+    display: flex;
+    width: 330px;
+    height: 48px;
+    font-size: 15px;
+  }
+
+  #firstname {
+    margin: auto;
+    border: 1px solid #d0d9d4;
+    border-radius: 4px;
+  }
+
+  .frame-form #firstname {
+    display: flex;
+    width: 330px;
+    height: 48px;
+    font-size: 15px;
+  }
+
+  .frame-form li {
+    list-style: none;
+  }
+}
+
+@media only screen and (min-width: 500px) and (max-width: 1800px) {
```

Comment:
Well done on making some of the changes requested by the previous reviewer 💯 but some of the changes you introduced broke the design a bit, take a look 👇🏽 

![Screenshot 2023-05-07 at 00 06 09](https://user-images.githubusercontent.com/45739963/236657075-890477fd-5419-4ec7-a6c5-60f4033295c2.png)

I think the issue is related to the fact that you're using `max-width` which makes that when the screen reaches a certain `width` the media query stops applying to it. ❌